addons/rose/package.json

```diff
@@ -96,12 +97,16 @@
     "jest": "^27.4.7",
     "loader.js": "^4.7.0",
     "npm-run-all": "^4.1.5",
+    "postcss": "^8.4.18",
```

Comment:
Why is PostCSS added?

Reply:
Curious about this too.

Reply:
Forgot to comment about this, stylelint 14 [does not support any non-css](https://stylelint.io/migration-guide/to-14#syntax-option-and-automatic-inferral-of-syntax) syntax out of the box anymore. 

Both [stylelint-config-standard-scss](https://github.com/stylelint-scss/stylelint-config-standard-scss/blob/main/package.json#L64) and [stylelint-config-sass-guidelines](https://github.com/bjankord/stylelint-config-sass-guidelines/blob/main/package.json#L38) have a peer dependency on `postcss` so they won't work without it.

Reply:
Ahh! that makes all the sense then 😉